volatility3/framework/plugins/linux/lsof.py

```diff
@@ -69,21 +92,75 @@ def list_fds(
             for fd_fields in fd_generator:
                 yield pid, task_comm, task, fd_fields
 
+    @classmethod
+    def list_fds_and_inodes(
+        cls,
+        context: interfaces.context.ContextInterface,
+        symbol_table: str,
+        filter_func: Callable[[int], bool] = lambda _: False,
+    ):
+        for pid, task_comm, task, (fd_num, filp, full_path) in cls.list_fds(
+            context, symbol_table, filter_func
+        ):
+            inode_metadata = cls.get_inode_metadata(filp)
```

Comment:
So I'd do a check for `None` here and then populate it with AbsentValues.  The function may get called as part of other computations, so returning None is easier to work with than a tuple of AbsentValues (which are really only supposed to be for the UI to know not to render anything, rather than to be passed around or used for computations).